backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2NativeCompilerArguments.kt

```diff
@@ -139,5 +140,10 @@ class K2NativeCompilerArguments : CommonCompilerArguments() {
     @Argument(value = "--verify_ir", description = "Verify IR")
     var verifyIr: Boolean = false
 
+    @Argument(value = "--embed_bitcode", description = "embed LLVM bitcode in object files")
```

Comment:
It makes sense to enable bitcode embedding by default on macOS and iOS targets.
@olonho WDYT?

Reply:
How much bigger binaries would become then?

Reply:
The smallish sample project I was testing with went from 153MB to 238MB